CHANGELOG.md

```diff
@@ -26,18 +26,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Config option `cursor.blink_timeout` to timeout cursor blinking after inactivity
 - Escape sequence to set hyperlinks (`OSC 8 ; params ; URI ST`)
 - Config `hints.enabled.hyperlinks` for hyperlink escape sequence hint highlight
+- On macOS, Alacritty now reads `AppleFontSmoothing` from user defaults to control font smoothing
 
 ### Changed
 
 - The `--help` output was reworked with a new colorful syntax
 - OSC 52 is now disabled on unfocused windows
 - `SpawnNewInstance` no longer inherits initial `--command`
 - Blinking cursor will timeout after `5` seconds by default
-- Deprecated `colors.search.bar`, use `colors.footer_bar` instead
+- Deprecate `colors.search.bar`; use `colors.footer_bar` instead
```

Comment:
This change should be reverted, the existing entry is perfectly fine and imo superior.

Reply:
I'll change it back, but for context, this wording confused me because messages like "Deprecated <foo>[, use <bar> instead]" appear elsewhere in the changelog in "Removed" sections, except those entries refer to the removal of the deprecated option, rather than the deprecation itself. The use of "deprecated" rather than "deprecate" makes it ambiguous because it's both a verb and an adjective, and you need the context of which section it's in to disambiguate. https://github.com/alacritty/alacritty/pull/6186#issuecomment-1179783098